src/ripple/app/misc/impl/TxQ.cpp

```diff
@@ -54,10 +54,9 @@ getFeeLevelPaid(ReadView const& view, STTx const& tx)
         return FeeLevel64(0);
     }
 
-    if (std::pair<bool, FeeLevel64> const feeLevelPaid =
-            mulDiv(effectiveFeePaid, TxQ::baseLevel, baseFee);
-        feeLevelPaid.first)
-        return feeLevelPaid.second;
+    if (auto const feeLevelPaid =
+            mulDiv(effectiveFeePaid, TxQ::baseLevel, baseFee))
+        return *feeLevelPaid;
 
     return FeeLevel64(std::numeric_limits<std::uint64_t>::max());
 }
```

Comment:
Can you use instead:

`return mulDiv(effectiveFeePaid, TxQ::baseLevel, baseFee).value_or(ripple::muldiv_max);`